### ISSUE TITLE
Launcher waits a window to be closed open another

### ISSUE THREAD

**Issue by IntrepidoC** [Healix/Gw2Launcher#54]
Hi, sorry if this is because I can`t configure the launcher correctly, but I always used another launcher and got to run 3-5 simultaneous games. I downloaded gw2launcher and it seems to be a superior option. I set up 2 of my accounts to test it, but the 2nd instance keeps "waiting" and just launches the game after the 1st instance is closed.

I already tried to run with the "maximum number of active accounts" option checked (max 5) and unchecked. The delay when lauching multiple accounts also checked and unchecked. Nothing worked.

**Comment by Healix** [comment]
Using d912pxy? It prevents multiple clients from starting. To use d912pxy on multiple accounts, you must enable "localized execution" set to "full" under Settings > Guild Wars 2 > Management.

Other than that, there's a few options that could cause it. If you choose to launch via "launch (normal)" that's how GW2 would normally launch, limited to 1 account at a time. The delay launching options (under Settings > General > Launching) would be the other cause, if any of them are enabled.

**Comment by IntrepidoC** [comment]
Thanks for the prompt response. I had d912pxy installed before, but uninstalled it (via addon manager) because gw2launcher > d912pxy, so I think this was not the issue. I also had the "localized execution" enabled thw whole time.

I really launched using "launch (normal)" because I thought that was the proper way. After your responde, I tried to launch without the normal option and the 2nd account just became "failed". Any other insghts?

**Comment by Healix** [comment]
Hover over the failed account and it'll give a reason.

Try the beta version:
https://github.com/Healix/Gw2Launcher/raw/master/Gw2Launcher/bin64/Beta/Gw2Launcher.exe

**Comment by IntrepidoC** [comment]
"Failed to create localized path:
O acesso ao caminho 'D:\Guild Wars 2\Gw2Launcher\2\Gw2-64.exe' foi negado"
the second sentence, which is in portuguese, menas something like "the access to path ... was denied"

I can't try the beta now, but if you can't identify the error until the evening, I'll give it a try.

**Comment by Healix** [comment]
The beta version has a change related to that error. If that doesn't fix it, Gw2Launcher is likely being blocked by other software.

Deleting that Gw2Launcher folder and changing localized execution to "full" and "core" should temporarily fix the problem. If you're not using d912pxy and don't need "full" localized execution, changing it to "binaries" will also avoid the problem.

**Comment by IntrepidoC** [comment]
I'll give that a try and return to you later at night, since it is still morning here. Thanks for your attention.

**Comment by IntrepidoC** [comment]
It worked deleting the Gw2Launcher folder and changing localized execution to binaries (since I don't use d912pxy anymore). Don't know if any change would solve it by itself because I did them both at once. I'll refrain from using the beta version because I'm not that tech savvy and I can wait for it to be launched as a stable  version. I was able to launch all my 4 alt accounts at once side by side.

Thank you so much for the help and for making Gw2Launcher.
